validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+
+### Alert Manager Integration
+
+Test Longhorn alert integration with Rancher Alert Manager for storage-related events and notifications.
+
+| Step Number | Action | Data | Expected Result                           |
+|------|--------|--------|--------------------------------------------|
+| 1 | Verify Rancher monitoring and alerting are configured | | Alert Manager operational                  |
+| 2 | Navigate to "Monitoring" → "Alertmanager" in Rancher | | Alert Manager interface accessible         |
+| 3 | Verify Longhorn alert rules exist in Prometheus rules | | Rules for volume/node alerts present       |
+| 4 | Configure notification channel (Slack/email) for testing | | Notification channel configured            |
+| 5 | Simulate volume degraded state by shutting down node | | Node shutdown triggers volume degradation  |
+| 6 | Verify alert appears in Alert Manager within 2 minutes | | `LonghornVolumeStatusCritical` alert fires |
+| 7 | Check notification delivery through configured channel | | Alert notification received                |
+| 8 | Simulate storage space warning by filling disk | | Disk usage exceeds threshold               |
+| 9 | Verify storage warning alert triggers | | `LonghornDiskStorageWarning` alert fires   |
+| 10 | Resolve issues and verify alert resolution | | Alerts automatically resolve               |
+| 11 | Confirm resolution notifications sent | | Resolution notifications received          |
+
+### Backup Target Configuration
+
+Test backup target setup and configuration for S3 and NFS storage through Rancher interface.
+
+| Step Number | Action | Data | Expected Result                    |
+|------|--------|--------|-------------------------------------|
+| 1 | Access Longhorn Settings through Rancher → Longhorn | | Longhorn settings page displays     |
+| 2 | Configure S3 backup target: `s3://test-bucket@us-west-2/longhorn/` | | S3 backup target configured         |
+| 3 | Create Kubernetes secret with S3 credentials | | Secret created with AWS access keys |
+| 4 | Set "Backup Target Credential Secret" to created secret | | Credential secret configured        |
+| 5 | Click "Test Connection" for backup target | | Connection test passes successfully |
+| 6 | Create test volume with sample data | | Volume created and data written     |
+| 7 | Navigate to Volume in Longhorn UI and click "Create Backup" | | Backup creation dialog opens        |
+| 8 | Create backup with name "test-backup-001" |  | Backup process initiates            |
+| 9 | Monitor backup progress until completion | | Backup completes successfully       |
+| 10 | Verify backup appears in backup list | | Backup listed with correct metadata |
+| 11 | Test backup integrity verification | | Backup integrity check passes       |
+
+### Cross-Cluster Disaster Recovery
+
+Test disaster recovery volume functionality between separate Rancher-managed clusters with shared backup storage.
```

Comment:
this test should list the prerequisites clearly in the steps of the test. i.e. do you want 2 vanilla downstream clusters before running? Should they already have longhorn installed? If so, should the backup(s) already be configured? etc.

Reply:
done